### DISCUSSION TITLE
Joomla 5 Template

### DISCUSSION THREAD

**Discussion by gochitashvili** [joomla/joomla-cms#42532] [Q&A]
Hello everyone, where I can find documentation about Joomla 5 template development?

**Comment by sayfrndship1234** [comment]
Nice question !

**Comment by sba** [comment]
[https://docs.joomla.org/Portal:Joomla_5#Joomla.21_Templates](https://docs.joomla.org/Portal:Joomla_5#Joomla.21_Templates)
It was written for Joomla 4, but not much has changed for Joomla 5 template development
